analysed-packages/tcpdump/version-4.99.1/tcpdump-4.99.1-SPDX2TV.spdx

```diff
@@ -0,0 +1,19385 @@
+Copyright (C) 1995, 1996, 1997, and 1998 WIDE Project. All rights reserved. </text>
+
+##File
+
+FileName: tcpdump-4.99.1.tar.gz/tcpdump-4.99.1.tar/tcpdump-4.99.1/print-snmp.c
+SPDXID: SPDXRef-item1161896
+FileChecksum: SHA1: 96404dabd57c3040cf4722028797994b12dd3fad
+FileChecksum: SHA256: b504184591baba7be5b20aa89b2228279fb5e09eb1bb1df161778e0706ffbc3c
+FileChecksum: MD5: 0a76c2b2c3867f234ad568860b76358b
+LicenseConcluded: LicenseRef-BSD-2-Clause-55ee54facd173f8169b9a4625f2f79ec AND LicenseRef-Permission-Notice-3a910945594b4de572e0943feee55bcb
+LicenseInfoInFile: LicenseRef-Less
+LicenseInfoInFile: LicenseRef-BSD-2-Clause
+FileCopyrightText: <text> Copyright (c) 1990, 1991, 1993, 1994, 1995, 1996, 1997 John Robert LoVerso. All rights reserved.
+Copyright (c) 1999.
+Copyright (c) 1990, 1991, 1993, 1994, 1995, 1996, 1997 This software was produced under a U.S. Government contract W-7405-ENG-36) by Los Alamos National Laboratory, which is operated by the	University of California for the U.S. Department of Energy.  </text>
+
+##File
+
+FileName: tcpdump-4.99.1.tar.gz/tcpdump-4.99.1.tar/tcpdump-4.99.1/print-enc.c
+SPDXID: SPDXRef-item1163239
+FileChecksum: SHA1: 1424350f9cb4eeef42da0b897747491b20e35b1c
+FileChecksum: SHA256: c5a8d6de936589748a181111b514bd4393ce7ec647df1fc15941ffacf0b66bf8
+FileChecksum: MD5: b261dff3f15cf19a89b8d3443ef6bd2b
+LicenseConcluded: LicenseRef-BSD-4-Clause-Shortened-e96ddab5f8fffe47de17b5812d71176a AND LicenseRef-Permission-Notice-3a2bf8a18b88925f22a9130d4cf995f0 AND LicenseRef-warranty-disclaimer-88394e5be8e0e24e8312d81013827b8f
+LicenseInfoInFile: LicenseRef-W3C-style
+LicenseInfoInFile: LicenseRef-BSD-4-Clause-Shortened
+FileCopyrightText: <text> Copyright (c) 1990, 1991, 1993, 1994, 1995, 1996 The Regents of the University of California. All rights reserved.
+Copyright (C) 1995, 1996, 1997, 1998 by John Ioannidis, Angelos D. Keromytis and Niels Provos.
+Copyright (c) 2001, Angelos D. Keromytis. </text>
+
+##File
+
+FileName: tcpdump-4.99.1.tar.gz/tcpdump-4.99.1.tar/tcpdump-4.99.1/install-sh
+SPDXID: SPDXRef-item1161774
+FileChecksum: SHA1: 3a246c73bb44f22ae59518509a4d8d7b5f66b126
+FileChecksum: SHA256: 30b6dfee94ddf9e36b85a8c280826787f34a87529674ff93a16d35026f3d0dda
+FileChecksum: MD5: 3b6f51f33692a0032c68885f75ffe342
+LicenseConcluded: LicenseRef-NTP-797f339d60a91c2001139ad5582f2953
+LicenseInfoInFile: LicenseRef-MIT
+LicenseInfoInFile: LicenseRef-NTP
+FileCopyrightText: <text> Copyright 1991 by the Massachusetts Institute of Technology </text>
+
+##File
+
+FileName: tcpdump-4.99.1.tar.gz/tcpdump-4.99.1.tar/tcpdump-4.99.1/print-stp.c
+SPDXID: SPDXRef-item1161764
+FileChecksum: SHA1: 2c82d77ca8631e9fc51f59f06715980aafed999e
+FileChecksum: SHA256: b6752c230fcc29bb30c785f5dddd6e881517695c82953ad82e7af3cd2bec3dba
+FileChecksum: MD5: fa8e7659ce06db01279bc69a53eb3340
+LicenseConcluded: LicenseRef-BSD-3-Clause-499bc2ee779fcca8f326572ae23b93fd OR LicenseRef-GPL-1.0-only
```

Comment:
the conclusion in this case is GPL-1.0-or-later not "-only" because the GPL says if no version number of GPL is specified the user is free to choose one among the official released GPL versions, due to this we always conclude GPL-1.0-or-later in all such cases